docs/how-to-guides/cleanup/clean-up-aws-cloudwatch-alarms.mdx

```diff
@@ -42,29 +42,68 @@ This guide assumes that you have already [installed](../../getting-started/insta
 
    :::
 
-3. Update the `plugin_cleanup_aws_alarms` section with the desired target cloud account IDs and setting the `enabled` property to `true`:
-
-   ```yaml title="cleanup_aws_alarms plugin configuration"
-   plugin_cleanup_aws_alarms:
-     # Dictionary of key cloud with list of account IDs for which the plugin should be active as value
-     config:
-       aws:
-   # highlight-start
-         - '1234567'
-         - '567890'
-   # highlight-end
-     # Enable plugin?
-   # highlight-next-line
-     enabled: true
+3. Use the [`app install` command](../../reference/cli/app/install.mdx) to install the `cleanup-aws-alarms` app:
+
+   ```bash
+   > app install cleanup-aws-alarms
+   ```
+
+   :::info
+
+   Fix Inventory will create a default config `fix.apps.cleanup_aws_alarms`.
+
+   :::
+
+4. Execute the following command in [Fix Inventory Shell](../../reference/components/shell.mdx) to open the infrastructure app configuration for editing:
+
+   ```bash
+   > config edit fix.apps.cleanup_aws_alarms
+   ```
+
+5. Update the configuration with the desired target cloud account IDs:
+
+   ```yaml title="cleanup_aws_alarms configuration"
+   clouds_and_accounts:
+     aws:
+       - '1234567'
+       - '567890'
+   ```
+
+6. Run the app using the [`app run` command](../../reference/cli/app/run.mdx):
+
+   ```bash
+   > app run cleanup-aws-alarms
    ```
 
-The plugin will now run each time Fix Inventory emits the `post_cleanup_plan` [event](../../reference/events/index.mdx). The `post_cleanup_plan` event is emitted in the [`cleanup` phase](../../reference/workflows/index.mdx#cleanup) of the [`collect_and_cleanup` workflow](../../reference/workflows/index.mdx#collect_and_cleanup-workflow).
+   :::tip
+
+   Add the optional `--dry-run` flag to see what commands the app would perform, without actually executing them.
+
+   :::
+
+   :::note
+
+   Items tagged with `expiration: never` will not be flagged for cleanup.
+
+   :::
+
+7. [Create an event-based job](../automation/create-an-event-based-job.mdx) to run the app automatically:
+
+   ```bash
+   > jobs create --name "Clean Up AWS Alarms" --wait-for-event cleanup_plan 'app run cleanup-aws-alarms'
+   ```
+
+   :::info
+
+   The [`cleanup-aws-alarms` infrastructure app](../../reference/infrastructure-apps/cleanup-aws-alarms.mdx) will now run each time Fix Inventory emits the `post_cleanup_plan` [event](../../reference/events/index.mdx). The `post_cleanup_plan` event is emitted in the [`cleanup` phase](../../reference/workflows/index.mdx#cleanup) of the [`collect_and_cleanup` workflow](../../reference/workflows/index.mdx#collect_and_cleanup-workflow).
```

Comment:
The command line registers for `cleanup_plan`, but this info bubble says it would run on `post_cleanup_plan`.